bin/functions/setup.sh

```diff
@@ -62,7 +63,38 @@ function setup_db () {
     setup_db_do_migrations
 }
 
+# Schemas are managed at https://git.earthdata.nasa.gov/scm/emfd/otherschemas.git
+# and need to be imported manually into CMR. This function will distribute schemas
+# to all the applications that need to consume these schemas and their related files
+# to support the Generic Document Pipeline workflow.
+# Note: Schemas may be published with any directory case but the schemas are
+# consumed in lower case by CMR code.
+function setup_schemas () {
+  apps=("search-app"
+              "ingest-app"
+              "indexer-app"
+              "metadata-db-app"
+              "system-int-test")
```

Comment:
White space isn't consistent here

Reply:
I have removed this function. it is no longer needed.